lib/stage0-handoff/src/rot_update_details.rs

```diff
@@ -19,7 +19,7 @@ unsafe impl HandoffData for RotBootState {
 ///
 /// It gets read from RAM by the `lpc55-update-server`
 #[derive(
-    Debug, Clone, PartialEq, Eq, Deserialize, Serialize, SerializedSize,
+    Debug, Copy, Clone, PartialEq, Eq, Deserialize, Serialize, SerializedSize,
```

Comment:
(this type is of nontrivial size -- 92 bytes, if my head-compiler is working correctly -- so be sure you actually want `Copy` here, as it may produce hidden copies with measurable performance impact.)